lib/broadway_cloud_pub_sub/producer.ex

```diff
@@ -56,6 +64,23 @@ defmodule BroadwayCloudPubSub.Producer do
 
   The above configuration will set up a producer that continuously receives messages
   from `"projects/my-project/subscriptions/my_subscription"` and sends them downstream.
+
+  ## Acking
+
+  You can use the `:on_success` and `:on_failure` options to control how messages are acked on PubSub.
+  By default successful messages are acked and failed messages are ignored.
+  You can set `:on_success` and `:on_failure` when starting the PubSub producer,
+  or change them for each message through `Broadway.Message.configure_ack/2`
+
+  Here is the list of all possible values supported by `:on_success` and `:on_failure`:
+
+  * `:ack` - Acknowledge the message. PubSub will mark the message as acked.
+  * `:ignore` - Don't do anything. It won't notify to PubSub, and it will apply the default deadline.
+  * `:no_ack` - Change the deadline to 0 seconds.
```

Comment:
What do you think about using `:nack` instead of `:no_ack`?  Some of the other Pub/Sub implementations use this term explicitly, and I wonder if we might do the same for consistency. 

For reference:
 - Go: https://godoc.org/cloud.google.com/go/pubsub#Message.Nack
 - Python: https://googleapis.dev/python/pubsub/latest/subscriber/index.html#explaining-ack

FWIW, the Pub/Sub docs make no mention of nack, so this is open to interpretation.

Reply:
I think that it's great to have some consistency with the other libraries, as you say it's not really an action in pubsub, it's just telling PubSub to change the deadline for that message, but it can be used as the nack of other message brokers.